front/app/components/ParticipationCTABars/BudgetingCTABar/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState, FormEvent } from 'react';
+
+// Components
+import { Button } from '@citizenlab/cl2-component-library';
+import { ParticipationCTAContent } from 'components/ParticipationCTABars/ParticipationCTAContent';
+
+// hooks
+import { useTheme } from 'styled-components';
+import useBasket from 'hooks/useBasket';
+
+// services
+import { IPhaseData, getCurrentPhase, getLastPhase } from 'services/phases';
+
+// utils
+import { scrollToElement } from 'utils/scroll';
+import {
+  CTABarProps,
+  hasRrojectEndedOrIsArchived,
+} from 'components/ParticipationCTABars/utils';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../messages';
+
+export const BudgetingCTABar = ({ phases, project }: CTABarProps) => {
```

Comment:
I notice it sets to "You have participated in this project" after you add things to your basket (not after you submit your basket). Just want to check if that's correct behaviour??

![image](https://user-images.githubusercontent.com/33987955/213697590-ca96d44a-8435-4b2e-86d1-049c7d519a11.png)